src/D2L.Bmx/ConsolePrompter.cs

```diff
@@ -84,7 +84,8 @@ internal class ConsolePrompter : IConsolePrompter {
 			readKey = () => (char)_stdinReader.Read();
 		}
 
-		Console.Error.Write( $"{ParameterDescriptions.Password}: " );
+		string passwordPrompt = string.Format( ParameterDescriptions.Password, user, org );
+		Console.Error.Write( $"{passwordPrompt}: " );
```

Comment:
A single line is getting long. Not super readable at a glance.

Let's do it like this
```csharp
Console.Error.Write( $"{ParameterDescriptions.Org}: {org}" );
Console.Error.Write( $"{ParameterDescriptions.User}: {user}" );
Console.Error.Write( $"{ParameterDescriptions.Password}: " );
```

This also keeps all parameter descriptions unchanged.

Reply:
> Let's do it like this

Sure